CHANGELOG.md

```diff
@@ -35,6 +35,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v0.37.13] - 2020-05-29
```

Comment:
Isn't this supposed to be in a separate file?

Reply:
I didn't backport the ci update, the 0.37 line still has all the old ci. do you think I should?

Reply:
Nahhh, don't worry about it.